src/models/pretrained.jl

```diff
@@ -40,3 +40,22 @@ function load_fashion_mnist_vae(; strong=true)
     end
     return vae
 end
+
+function load_cifar_10_mlp()
+    M = deserialize(joinpath(vision_dir, "cifar_10_mlp.jls"))
+    return M
+end
+
+function load_cifar_10_ensemble()
+    M = deserialize(joinpath(vision_dir, "cifar_10_ensemble.jls"))
+    return M
+end
+
+function load_cifar_10_vae(; strong=true)
+    if strong
+        vae = deserialize(joinpath(vision_dir, "cifar_10_vae_strong.jls"))
+    else
+        vae = deserialize(joinpath(vision_dir, "cifar_10_vae_weak.jls"))
+    end
+    return vae
```

Comment:
Is vae defined here? it seems to me that it is declared in the if statement, which is one level lower

Reply:
Yes, it is quite unintuitive but it is defined. I'm attaching a snippet from my REPL to ensure you it works that way:

``` 
julia> condition = true
true

julia> if condition
           x = 1
       else
           x = 0
       end
1

julia> x
1
```

Reply:
completely ridiculous